anime-relations.txt

```diff
@@ -143,9 +152,15 @@
 # Concrete Revolutio: Choujin Gensou -> ~ - The Last Song
 - 31147|11165:14-24 -> 32313|11620:1-11!
 
+# D-Frag! -> ~ OVA
+- 20031|8001:13 -> 23383|8414:1
```

Comment:
The OVA doesn't take place after the TV series, so it shouldn't be the 13th episode.